packages/xo-web/src/xo-app/settings/config/index.js

```diff
@@ -60,7 +63,8 @@ export default class Config extends Component {
 
     return (
       <div>
-        <div className='mb-1'>
+        {getXoaPlan() !== SOURCES && <CloudConfig />}
```

Comment:
Is it available in all plans? /cc @olivierlambert

Reply:
Yes